From 4bf5a2a56c93ac15a47f3b4e0d0156eec9a1eb5d Mon Sep 17 00:00:00 2001
From: "Nguyen Van Linh [FGA.BU13]" <LinhNV25@fsoft.com.vn>
Date: Mon, 9 Apr 2018 18:54:02 +0700
Subject: [PATCH 237/628] mmc: core: host: add property cap-mmc-wait-busy

enable device tree to set MMC_CAP_WAIT_WHILE_BUSY
by adding cap-mmc-wait-busy property

Signed-off-by: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNV25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 drivers/mmc/core/host.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mmc/core/host.c b/drivers/mmc/core/host.c
index da950c4..0388b47 100644
--- a/drivers/mmc/core/host.c
+++ b/drivers/mmc/core/host.c
@@ -255,6 +255,8 @@ int mmc_of_parse(struct mmc_host *host)
 		host->caps |= MMC_CAP_SD_HIGHSPEED;
 	if (of_property_read_bool(np, "cap-mmc-highspeed"))
 		host->caps |= MMC_CAP_MMC_HIGHSPEED;
+	if (of_property_read_bool(np, "cap-mmc-wait-busy"))
+		host->caps |= MMC_CAP_WAIT_WHILE_BUSY;
 	if (of_property_read_bool(np, "sd-uhs-sdr12"))
 		host->caps |= MMC_CAP_UHS_SDR12;
 	if (of_property_read_bool(np, "sd-uhs-sdr25"))
-- 
2.7.4

